.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,17 @@
+<!-- Put a description of what this PR is for here, along with any references to issues that this resolves or contributes to -->
+
+
+
+
+<!-- Replace each [ ] with [X] to check it. Switch to the preview view to make it easier to click on links -->
+If this PR is for Terraform, I acknowledge that I have:
+
+- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
+- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
+- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
+- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
+- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.
```

Comment:
Do community contributors add release notes? I thought that part was for the reviewer in those cases.

Reply:
We changed it: https://github.com/GoogleCloudPlatform/magic-modules/pull/3456. I think it was because the CI will fail if the release note isn't correctly formatted, which led to people being confused why their code is failing (since they can't see cloud build results).